README.md

```diff
@@ -8,50 +8,203 @@ https://readthedocs.org/projects/actually-robust-training/badge/?version=latest&
+    model_class = YourLightningModule
+
+    project = ArtProject(name="quickstart", datamodule=datamodule)
+
+    project.add_step(CheckLossOnInit(model_class))
+    project.add_step(OverfitOneBatch(model_class))
+    project.add_step(Overfit(model_class))
+
+    project.run_all()
+```
+
+As a result, if you logged any metrics with self.log, you should observe something like this:
+
+```
+Summary:
+Step: Check Loss On Init, Model: YourLightningModule, Passed: True. Results:
+         loss-valid: 16.067102432250977
+         accuracy-train: 0.75
+Step: Overfit One Batch, Model: YourLightningModule, Passed: True. Results:
+         loss-train: 0.00035009183920919895
+         accuracy-train: 0.800000011920929
+Step: Overfit, Model: YourLightningModule, Passed: True. Results:
+         loss-train: 2.1859991550445557
+         accuracy-train: 0.75
+         loss-valid: 3.5187878608703613
+```
+
+[Explore all available Steps](not-existing-link)
+
+### Adding checks
+Ideally, each step should be accompanied by a set of _checks_. ART will not move to the next step without passing checks from previous steps.
+
+```python
+    from art.steps import CheckLossOnInit, Overfit, OverfitOneBatch
+
+    datamodule = YourLightningDataModule()
+    model_class = YourLightningModule
+
+    project = ArtProject(name="quickstart", datamodule=datamodule)
+
+    # Let's assume we are working on a typical classification problem
+    NUM_CLASSES = 10
+    EXPECTED_LOSS = -math.log(1 / NUM_CLASSES) # 2.3
+
+    project.add_step(
+        CheckLossOnInit(model_class),
+        #You must know logged metric names
+        checks=[CheckScoreCloseTo("loss-valid", EXPECTED_LOSS, rel_tol=0.01)],
+    )
+    project.add_step(
+        OverfitOneBatch(model_class),
+        checks=[CheckScoreLessThan("loss-train", 0.001)],
+    )
+    project.add_step(
+        Overfit(model_class),
+        checks=[CheckScoreLessThan("accuracy-train", 0.9)],
+    )
+
+    project.run_all()
+```
+
+This time, depending on your scores, you may observe something like this:
+
+```
+Check failed for step: Check Loss On Init. Reason: Score 16.067102432250977 is not equal to 2.3
+Summary:
+Step: Check Loss On Init, Model: YourLightningModule, Passed: False. Results:
+         loss-valid: 16.067102432250977
+         accuracy-train: 0.75
+```
+
+### Debug your Neural Network
+Track network evolution, gradient values, and save images by decorating your functions
+```python
+    from art.decorators import BatchSaver, art_decorate
+
+    datamodule = YourLightningDataModule()
+    model_class = YourLightningModule
+    art_decorate([(model_class, "forward")], BatchSaver())
+
+    project = ArtProject(name="quickstart", datamodule=datamodule)
+    # For this to work we should allow providing empty checks. And just print a warning... man use checks.
+    project.add_step(CheckLossOnInit(model_class))
+    project.run_all()
+
+```
+
+### Get control over what is being calculated
```

Comment:
Also it may be too huge for quickstart? Just loudly thinking, we introduce it in tutorials.